src/microbe_stage/IEngulfable.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+
+/// <summary>
+///   Objects that can be engulfed by a microbe.
+/// </summary>
+[UseThriveSerializer]
+public interface IEngulfable : IGraphicalEntity
+{
+    /// <summary>
+    ///   The size of this engulfable object based on microbe hex count.
+    /// </summary>
+    public float Size { get; }
```

Comment:
I think something like EngulfSize might make more sense here? At least to me having Size and Radius next to each other where the Size seems to be a gameplay stat and Radius is actually a graphical size related thing, seems pretty prone to confusion to me.